src/libraries/System.Threading.ThreadPool.WebAssembly.Threading/src/System.Threading.ThreadPool.WebAssembly.Threading.csproj

```diff
@@ -0,0 +1,8 @@
+<Project Sdk="Microsoft.Build.Traversal">
```

Comment:
What's the reason to have this file? Packaging doesn't require it.

Reply:
Without this, I wasn't getting a nuget in `artifacts/packages/` if I just did `./build.sh --os browser -c Release` (both with or without `/p:WasmEnableThreading`).

Reply:
I don't think you need these Traversal projects at all anymore. In the past they were required but they shouldn't be anymore.